atd-vze/src/views/Locations/Location.js

```diff
@@ -111,7 +111,7 @@ function Location(props) {
             notes={data?.location?.location_notes}
             INSERT_NOTE={INSERT_LOCATION_NOTE}
             UPDATE_NOTE={UPDATE_LOCATION_NOTE}
-            DELETE_NOTE={DELETE_LOCATION_NOTE}
+            SOFT_DELETE_NOTE={DELETE_LOCATION_NOTE} // TODO actually make location notes use a soft delete
```

Comment:
we are writing up a backlog issue to add soft deletes for location notes

Reply:
yes we are!
https://github.com/cityofaustin/atd-data-tech/issues/19046